packages/apt/test.sh

```diff
@@ -185,6 +187,26 @@ if [ "${can_upgrade}" = "yes" ]; then
   run_test
 fi
 
+echo "::group::Downgrade"
+
+if [ "${can_downgrade}" = "yes" ]; then
+  pgroonga_current_version=$(apt-cache search --full \^${pgroonga_package}\$ | \
+                               grep Version | \
+                               sed -E 's/^Version: ([0-9]\.[0-9]\.[0-9])-[0-9]/\1/')
+  createdb downgrade
+  psql downgrade -c 'CREATE EXTENSION pgroonga'
+  psql downgrade -c "'ALTER EXTENSION pgroonga UPDATE TO \'${pgroonga_current_version}\''"
```

Comment:
Maybe we can write it like the following.
```suggestion
  psql downgrade -c "ALTER EXTENSION pgroonga UPDATE TO '${pgroonga_current_version}'"
```